components/BlogPostCards.vue

```diff
@@ -44,24 +56,19 @@
     </div>
   </div>
   <div v-else class="h-screen">
-    <p> Loading .... </p>
+    <p>Loading ....</p>
   </div>
 </template>
 
 <script>
 export default {
-  data() {
-    return {
-      isLoaded: false,
-    };
-  },
   props: {
     posts: {
       type: Array,
-      required: true,
-    },
-    // In order to use this component the posts prop must be passed in. The contents of posts is the required fields from a graphql call. 
-    // The structure to make this call should be similar to this... 
+      required: true
+    }
+    // In order to use this component the posts prop must be passed in. The contents of posts is the required fields from a graphql call.
+    // The structure to make this call should be similar to this...
     // async getData() {
```

Comment:
If the commented code isn't being used, it should be removed.  If this is a necessary as an example, it should be moved into the developer documentation